tests/moment-timezone/guess.js

```diff
@@ -0,0 +1,115 @@
+"use strict";
+
+var tz = require("../../").tz;
+
+var getTimezoneOffset = Date.prototype.getTimezoneOffset;
+
+function mockTimezoneOffset (jan, jul) {
+	Date.prototype.getTimezoneOffset = function () {
+		var month = this.getMonth();
+		if (month > 3 && month < 9) {
+			return -jul;
+		}
+		return -jan;
+	};
+}
+
+function makeGuessTest (janOffset, julOffset, zone) {
+	return function (test) {
+		mockTimezoneOffset(janOffset, julOffset);
+		tz._guess = null;
+		test.equal(tz.guess(), zone);
+		test.done();
+	};
+}
+
+exports.guess = {
+	tearDown : function (done) {
+		Date.prototype.getTimezoneOffset = getTimezoneOffset;
+		done();
+	},
+
+	// These tests were built with 2015b data.
```

Comment:
Were they built manually or with a script? If a script was written, could you also share it to have the ability to rebuild when `moment` updates with the new data?